### PR TITLE
Add mytank page with picture upload

### PR DESCRIPTION
**Pull request by hennifant**

### CODE REVIEW

File: src/components/TankCard.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components';
+
+export default function Card({ deleteTank, title, text, fish, img }) {
+  return (
+    <CardContainer>
+      <h1>{title}</h1>
```
Comment (mwarnck) [inline src/components/TankCard.js:6]: h1 is not the right heading level I think when there could be more than one tank card rendered on this page

**Review by mwarnck — COMMENTED** [review]
Overall nice work. I left a few comments you should have a look at :)

File: src/App.js [hunk 2]
```diff
@@ -74,6 +92,10 @@ function App() {
   function handleChangeFilterBookmark(value) {
     setNewFilterBookmark(value);
   }
+
+  function deleteTank(id) {
+    setNewTank(newTank.filter(note => note.id !== id));
```
[the remark this answers is not in the document]
Comment (hennifant) [reply src/App.js:97]: You're absolutely right, thanks for the hint.

File: src/components/CreateTankForm.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  useEffect(() => {
+    setInputData({ ...inputData });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const handleFormSubmit = event => {
+    event.preventDefault();
+    setNewTank([inputData, ...newTank]);
+    setInputData({ title: '', text: '', fish: '' });
+    navigate('./');
```
[the remark this answers is not in the document]
Comment (hennifant) [reply src/components/CreateTankForm.js:29]: As you suggested, the -1 worked - thanks!

File: README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 ## Capstone Project of the IT bootcamp at neuefische
 
-Tank Buddy is under construction
+AppQuarium is under construction
```
[the remark this answers is not in the document]
Comment (hennifant) [reply README.md:5]: Thank you very much for all the hints - I have learned a lot from it. :-)

**Pull request merged**